src/Util.ts

```diff
@@ -35,4 +35,11 @@ export const getFrameTextSize = (configSize: number, textLength: number) => {
         throw new Error('Frame text should be between 0 to 30 characters')
     }
     return configSize / factor;
-};
\ No newline at end of file
+};
+
```

Comment:
this function is not readable to me and one more thing, why is the default value 7 any reason for that ?